solidity/contracts/system/TBTCSystem.sol

```diff
@@ -143,18 +140,15 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
             _keepThreshold,
             _keepSize
         );
-        vendingMachine = _vendingMachine;
         setTbtcDepositToken(_tbtcDepositToken);
         initializedTimestamp = block.timestamp;
         allowNewDeposits = true;
     }
 
     /// @notice Returns whether new deposits should be allowed.
-    /// @return True if new deposits should be allowed, both by the emergency pause button
-    ///         and respected the max supply schedule.
+    /// @return True if new deposits should be allowed by the emergency pause button
```

Comment:
“Should“ is pretty funny phrasing considering that if this returns false, new deposits *are* not allowed :grin: But I think we were already using that anyway.